$NetBSD$

(1) This patch is reference only, not actually used.
(2) automake on this directory won't be run with usual make package
    This may be used to generate Makefile.in file

--- work/tcode/tc/lisp/Makefile.am.orig	2012-07-10 13:12:21.000000000 +0900
+++ ./lisp/Makefile.am	2012-07-10 14:08:19.000000000 +0900
@@ -10,9 +10,9 @@
 	tc-inst.el tc-stat.el tc-ja-alnum.el \
 	try-tbl.el try-etbl.el ttc-tbl.el ttc-rtbl.el ttc-ltbl.el
 
-lisp_LISP = tc-sysdep.el $(COMMON_EL_FILES) tc-is18.el tc-is19.el tc-is20.el tc-is22.el \
+lisp_LISP =              $(COMMON_EL_FILES) tc-is18.el tc-is19.el tc-is20.el tc-is22.el \
 	tc-image.el tc-mkmzdic.el $(PREFILE)
-lisp_DATA = tc-bitmap.el
+lisp_DATA = tc-bitmap.el tc-sysdep.el
 ELCFILES = compile.log
 noinst_LISP = make-elc guess tc-pre-base
 
